### PR TITLE
heterogeneity todos

### PR DESCRIPTION
**Pull request by allaway**
### Purpose

To address todos in the heterogeneity section. 

### Directions for reviewers

Make sure my modifications make sense. I think i addressed the first two to-dos in this section by adding one additional sentence and removing one. 

#### Is there anything that you want to discuss further?

Are the references I selected for TODO number 3 OK? 

#### Is the pull request ready for review?

Yes 

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [x] All terms that need to be added to glossary have been added to #50 
- [x] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by allaway** [comment]
I also moved some periods.

**Comment by allaway** [comment]
Apologies for all of the builds. 😰

File: build/assets/custom-dictionary.txt [hunk 1]
```diff
@@ -122,7 +122,9 @@ Liu
 Pande
 pathologies
 Quellec
-underperformance
+undrperformance
```
Comment (jaclyn-taroni) [inline build/assets/custom-dictionary.txt]: Leftover from testing?

File: content/03.heterogeneity.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 These techniques can be applied to characterize imaging data [@doi:10.1016/j.media.2020.101660], mass cytometry data [@doi:10.1038/ncomms14825], ‘omics data, and others. 
-These methods not only help in reducing the number of features, but can also be used to visualize structure or artifacts in the data (e.g. [@doi:10.1038/s41467-019-13056-x]), define subgroups within data (e.g. [@doi:10.1038/s41467-020-15351-4], or for feature selection/extraction during application of specific machine learning models[@doi:10.1007/978-3-030-03243-2_299-1].
+These methods not only help in reducing the number of features, but can also be used to visualize structure or artifacts in the data (e.g. [@doi:10.1038/s41467-019-13056-x]), define subgroups within data (e.g. [@doi:10.1038/s41467-020-15351-4], or for feature selection/extraction during application of specific machine learning models.[@doi:10.1007/978-3-030-03243-2_299-1]
```
Comment (jaclyn-taroni) [inline content/03.heterogeneity.md:13]: Should the period be before or after the citation? We're inconsistent it looks like.

File: content/03.heterogeneity.md [hunk 3]
```diff
@@ -19,9 +23,9 @@ If non-biological heterogeneity is detectable, common approaches like  reprocess
...
-Putting constraints on the features learned by the model, through regularization, can help ensure that the learned representations are biologically relevant [TODO: REF]. 
-Low-dimensional representations trained on a collection of transcriptomic data can also be used as input to supervised machine learning methods[@doi:10.1186/s12859-020-3427-8; @doi:10.1016/j.procs.2016.07.014; @doi:10.1098/rsif.2017.0387]
+Representation learning through matrix factorization can extract composite features from transcriptomics datasets made of combinations of gene expression levels found in the training data that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], and use them to interpret test input data.[@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3] 
+Putting constraints on the features learned by the model, through regularization, can help ensure that the learned representations are generalizable and avoid overfitting of the resulting model [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782].
```
Comment (jaclyn-taroni) [inline content/03.heterogeneity.md:27]: Seems reasonable to me! (This _is_ TODO 3, correct?)

**Review by jaclyn-taroni — APPROVED** [review]
Overall LGTM. I had a question about punctuation (unfortunately) and we should fix the dictionary.

File: build/assets/custom-dictionary.txt (hunk 1 above)
Comment (allaway) [reply build/assets/custom-dictionary.txt]: No...just must have accidentally a letter!

Comment (allaway) [inline build/assets/custom-dictionary.txt]:
```suggestion
underperformance
```

File: content/03.heterogeneity.md (hunk 2 above)
Comment (allaway) [reply content/03.heterogeneity.md:13]: For superscript citations like Nature citation style I prefer to put the citation outside the punctuation (at least with a period...less sure about comma). This guide suggests always to put outside period or comma: https://libguides.usc.edu/c.php?g=293795&p=1956824

Nature Methods doesn't appear to have any specific guidance about this, however. 

What do you think?

File: content/03.heterogeneity.md (hunk 3 above)
Comment (allaway) [reply content/03.heterogeneity.md:27]: Yep, correct!

**Pull request merged**